src/cmd/filter.go

```diff
@@ -59,6 +76,34 @@ var listFilterCmd = &cobra.Command{
 	},
 }
 
+var updateFilterCmd = &cobra.Command{
+	Use:   "filter",
+	Short: "Update a filter",
+	Long: `Update a filter
+
+cat << EOF | opslevel update filter -f -
+id: "Z2lkOi8vb3BzbGV2ZWwvRmlsdGVyLzIzNTk"
+name: "Apps using RDS or DynamoDB"
+connective: "or"
+predicates:
+  - key: "tags"
+    keyData: "db"
+    type: "equals"
+    value: "rds"
+  - key: "tags"
+    keyData: "db"
+    type: "equals"
+    value: "dynamo"
+EOF`,
```

Comment:
```suggestion
	Long: `Update a filter`,
    Example: `
cat << EOF | opslevel update filter -f -
id: "Z2lkOi8vb3BzbGV2ZWwvRmlsdGVyLzIzNTk"
name: "Apps using RDS or DynamoDB"
connective: "or"
predicates:
  - key: "tags"
    keyData: "db"
    type: "equals"
    value: "rds"
  - key: "tags"
    keyData: "db"
    type: "equals"
    value: "dynamo"
EOF`,
```

Reply:
I also moved the the `ID` to the CLI command args instead of putting it into the yaml payload.  This has been the better practice lately because then you can share the Update and Create input types between create and update mutations.

This might require that we update opslevel-go to support this AND/OR paper over that in the CLI for now.